src/components/VariantSearchField.tsx

```diff
@@ -85,8 +86,12 @@ export const VariantSearchField = ({ onVariantSelect, currentSelection, triggerS
   );
   const [inputValue, setInputValue] = useState<string>('');
   const [menuIsOpen, setMenuIsOpen] = useState<boolean>(false);
-  const [variantQuery, setVariantQuery] = useState(currentSelection?.variantQuery ?? inputValue);
-  const [advancedSearch, setAdvancedSearch] = useState(currentSelection?.variantQuery !== undefined);
+
+  const [variantQuery, setVariantQuery] = useState(currentSelection?.variantQuery ?? '');
+  const [advancedSearch, setAdvancedSearch] = useState(!!currentSelection?.variantQuery);
+
+  //const [url, _] = useState<string>(window.location.href);
+  const exploreUrl = useExploreUrl();
```

Comment:
Hi @anastasia-escher, this line is a problem. The VariantSearchField should be pure (just like hopefully all other classes in the `components` folder) and only depend on the parameter input and not fetch data from `exploreUrl`.